scripts/bridge/populate_coach_umpire_bridge.py

```
@@ -0,0 +1,109 @@
+load_dotenv()
+
+
+def get_db_connection():
+    """Get PostgreSQL connection from environment variables."""
+    db_url = os.getenv("DATABASE_URL")
+    if db_url:
+        return psycopg2.connect(db_url)
+    else:
+        return psycopg2.connect(
+            host=os.getenv("PGHOST", "localhost"),
+            port=os.getenv("PGPORT", 5432),
```

Ruff (0.15.10)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)